en/micro-integrator/docs/references/connectors/ldap-connector/ldap-connector-example.md

```diff
@@ -0,0 +1,214 @@
+      "securityCredentials":"admin",
+      "secureConnection":"false",
+      "disableSSLCertificateChecking":"false",
+      "content":{ 
+         "objectClass":"identityPerson",
+         "dn":"uid=triss.merigold,ou=Users,dc=wso2,dc=org",
+         "attributes":{ 
+            "mail":"triss@wso2.com",
+            "userPassword":"geralt&triss",
+            "sn":"dim",
+            "cn":"dim",
+            "manager":"cn=geralt,ou=Groups,dc=example,dc=com"
+         }
+      }
+    }
+```
+
+2. Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here] 
+(https://curl.haxx.se/download.html).
+```
+curl -H "Content-Type: application/json" -X POST --data @student_data.json http://localhost:8290/student/create
+```
+
+**Expected Response**: 
+1. You should get a 'Success' response. 
+2. Open Apache Directory Studio and category DIT (Directory Information Tree) shows the hierarchical content of the 
+directory. Expand, collapse the tree and you will see the new entries. Select the entry and you will see it's attributes 
+and values on Entry Editor.
+![image](../../../assets/img/connectors/ldap_connector/ldap-connector-directory-studio-view.png)
+
+### Search ldap entries
+
+1. Create a file named search_student.json with following sample payload
+```json
+    {
+        "providerUrl": "ldap://localhost:10389/",
+        "securityPrincipal": "uid=admin,ou=system",
+        "securityCredentials": "admin",
+        "secureConnection": "false",
+        "disableSSLCertificateChecking": "false",
+        "application": "ldap",
+        "operation": "searchEntity",
+        "content": {
+            "objectClass": "identityPerson",
+            "filters": {
+                "manager": "cn=geralt,ou=Groups,dc=example,dc=com"
+            },
+            "dn": "ou=Users,dc=wso2,dc=org",
+            "attributes": "mail,uid"
+        }
+    }
+```
+
+2. Invoke the API as shown below using the curl command. 
+```
+curl -H "Content-Type: application/json" -X POST --data @search_student.json http://localhost:8290/student/search
+```
+
+**Expected Response**: 
+You should get all entries with match with the provided filter. A sample response is as follows.
+
+```json
+{
+    "result": {
+        "entry": [
+            {
+                "dn": "uid=triss.merigold,ou=Users,dc=WSO2,dc=ORG",
+                "mail": "triss@wso2.com",
+                "uid": "triss.merigold"
+            },
+            {
+                "dn": "uid=yennefer.of.vengerberg,ou=Users,dc=WSO2,dc=ORG",
+                "mail": "yenna@wso2.com",
+                "uid": "yennefer.of.vengerberg"
+            }
+        ]
+    }
+}
+```
+## What's Next
+
+* You can deploy and run your project on [Docker](../../../setup/installation/run_in_docker.md) or 
+[Kubernetes](../../../setup/installation/run_in_kubernetes.md).
+* To customize this example for your own scenario, see [File Connector Configuration](file-connector-config.md) 
```

Comment:
Let's remove this line for now. Will add it once the LDAP reference doc is done.

Reply:
Fixed